engine/source/materials/mat/mat069/sigeps69.F

```diff
@@ -134,46 +136,20 @@ SUBROUTINE SIGEPS69(
         GMAX  = GMAX  + MU(I)*AL(I)
       ENDDO
 C
+      GVMAX = ZERO
+      ETV   = ZERO
 C add viscosity using /visc/prony
       IF (IVISC > 0) THEN 
          NPRONY = INT(UPARVIS(1))
 c             
          DO I=1,NPRONY
           GI(I)    = UPARVIS(1 + I)
           BETA(I)  = UPARVIS(1 + NPRONY + I)
-          GMAX = GMAX + GI(I)
+          GVMAX = GVMAX + GI(I)
          ENDDO 
+         ETV = MIN(GVMAX,RBULK)/GMAX
```

Comment:
Is there a chance of division by 0 here? (can NPRONY be 0, or GI(i) be 0?)

Reply:
division is by GMAX not GVMAX